### PR TITLE
add README badge for CircleCI build

### PR DESCRIPTION
**Pull request by nuclearkatie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # trailmap
```
Comment (bam241) [inline README.md:1]: Maybe add a title corresponding to those badges.
like "Status" or ...

**Comment by nuclearkatie** [comment]
Done, thanks @bam241 for the suggestion! Good to merge...?

**Review by gonuke — APPROVED** [review]
LGTM! Thanks @nuclearkatie

**Review by bam241 — APPROVED** [review]
LGTM!

**Pull request merged**